geode-core/src/main/java/org/apache/geode/cache/client/internal/ServerRegionProxy.java

```diff
@@ -345,10 +345,11 @@ public List registerInterest(final Object key, final int interestType,
       final InterestResultPolicy policy, final boolean isDurable,
       final boolean receiveUpdatesAsInvalidates, final byte regionDataPolicy) {
     if (interestType == InterestType.KEY && key instanceof List) {
+      logger.warn(
+          "Register Interest on a List has been deprecated, an alternative is to use registerInterestForKeys(Iterable)");
```

Comment:
`Usage of registerInterest(List) has been deprecated. Please use registerInterestForKeys(Iterable)`

uniform like the one below?

Reply:
The reason why this one is different is because there is no API for registerInterest(List) but it just happened to be that they pass in a List and expect a certain type of behavior...